peekingduck/training/src/optimizers/adapter.py

```diff
@@ -0,0 +1,103 @@
+# Copyright 2023 AI Singapore
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+    Optimizer Class Adapter for Tensorflow
+
+    # @staticmethod
+    # def Adadelta(learning_rate: LearningRateSchedule, parameters):
+    #     raise NotImplementedError
+    #     return tf.keras.optimizers.Adadelta(learning_rate=learning_rate, **parameters)
+
+    # @staticmethod
+    # def Adafactor(learning_rate: LearningRateSchedule, parameters):
+    #     raise NotImplementedError
+    #     return tf.keras.optimizers.Adafactor(learning_rate=learning_rate, **parameters)
+
+    # @staticmethod
+    # def Adagrad(learning_rate: LearningRateSchedule, parameters):
+    #     raise NotImplementedError
+    #     return tf.keras.optimizers.Adagrad(learning_rate=learning_rate, **parameters)
+
+    @staticmethod
+    def Adam(learning_rate: LearningRateSchedule, parameters):
+        return .Adam(learning_rate=learning_rate, **parameters)
+
+    # @staticmethod # This is not released yet for tf-macos==2.10
+    # def AdamW(learning_rate: LearningRateSchedule, parameters):
+    #     raise NotImplementedError
+    #     # return tf.keras.optimizers.experimental.AdamW(learning_rate=learning_rate, **parameters)
+
+    # @staticmethod
+    # def Adamax(learning_rate: LearningRateSchedule, parameters):
+    #     raise NotImplementedError
+    #     return tf.keras.optimizers.Adamax(learning_rate=learning_rate, **parameters)
+
+    # @staticmethod
+    # def Ftrl(learning_rate: LearningRateSchedule, parameters):
+    #     raise NotImplementedError
+    #     return tf.keras.optimizers.Ftrl(learning_rate=learning_rate, **parameters)
+
+    # @staticmethod
+    # def Nadam(learning_rate: LearningRateSchedule, parameters):
+    #     raise NotImplementedError
+    #     return tf.keras.optimizers.Nadam(learning_rate=learning_rate, **parameters)
+
+    # @staticmethod
+    # def RMSprop(learning_rate: LearningRateSchedule, parameters):
+    #     raise NotImplementedError
+    #     return tf.keras.optimizers.RMSprop(learning_rate=learning_rate, **parameters)
+
+    @staticmethod
+    def SGD(learning_rate: LearningRateSchedule, parameters):
+        return tf.keras.optimizers.SGD(learning_rate=learning_rate, **parameters)
+
+
+"""
```

Comment:
Is above comment supposed to help user use the `Keras` optimizers? If yes, it should be state how to use. Otherwise, it should be removed.

Reply:
https://github.com/furyhawk/PeekingDuck/tree/47df8145fbba97081c6758cf66856b571d37db99

Removed comment.